resources/views/mship/management/dashboard.blade.php

```diff
@@ -367,7 +367,7 @@ class="btn btn-xs btn-danger">
                     <div class="row">
                         <div class="col-xs-12">
                             @if($_account->discord_id)
-                                Currently registered with Discord ID {{ $_account->discord_id }}. <br/>
+                                Currently registered with Discord ID {{ $_account->discord_user ? $_account->discord_user['username'].'#'.$_account->discord_user['discriminator'] : $_account->discord_id }}. <br/>
```

Comment:
You can rely on `$_account->discord_user` returning an correct here as it is within the else statement from line 369. `discord_user` only returns `null` if `discord_id` is blank looking at the getter on the model.

Reply:
Good catch  - https://github.com/VATSIM-UK/core/pull/2029/commits/bacf74390e917a543d3d37b3513e289f79bdac1a